setup.py

```diff
@@ -12,7 +12,7 @@ def readme():
 
 setup(
     name='pytest-astropy',
-    version='0.7.0',
+    version='0.8.0.dev',
```

Comment:
I would push this directly  to master as part of wrapping up the  release of yesterday rather than fold it into this PR, if you don't mind.

Reply:
Yeah, sure. I'll remove this.

Reply:
Lemme know if you need a squash. I was lazy and edited on GitHub directly.

Reply:
actually never mind, I see it wasn't customary in the past either to update the version number after a release :)

Reply:
OK, pushed.